player/public/coreplugins/org.sunbird.player.endpage-1.1/renderer/endpageApp.js

```diff
@@ -38,9 +40,11 @@ endPage.controller("endPageController", function($scope, $rootScope, $state,$ele
                     maxScore = maxScore + 0;
                 }
             });
-            $scope.score = ($scope.convert(totalScore) + "/" + $scope.convert(maxScore));
+            $scope.userScore = $scope.convert(totalScore);
+            $scope.totalScore = $scope.convert(maxScore);
         } else {
```

Comment:
Why you need else condition? already declared on top this two variables undefined

Reply:
Yes, the else is redundant. Pushed updated changes.
Please review.